push_hono_images.sh

```diff
@@ -35,10 +36,10 @@ then
     if [ -n "$CR" ]
     then
       IMAGE_NAME="$CR/$IMAGE_NAME"
-      docker tag $ECLIPSE_IMAGE_NAME:$TAG $IMAGE_NAME:$TAG
+      docker tag $ECLIPSE_IMAGE_NAME:$TAG $IMAGE_NAME:$NEWTAG
     fi
-    echo "pushing image $IMAGE_NAME:$TAG ..."
-    docker push $IMAGE_NAME:$TAG
+    echo "pushing image $IMAGE_NAME:$NEWTAG ..."
+    docker push $IMAGE_NAME:$NEWTAG
```

Comment:
what is this supposed to be good for?

Reply:
sorry, this shouldn't be in there. I use a slightly modified version of the push_hono_images to have the ability to push to our own registry when changes are not yet merged to the master branch. This shouldn't be in there...